src/devices/video/ef9365.cpp

```diff
@@ -776,105 +732,89 @@ int ef9365_device::get_char_pix( unsigned char c, int x, int y )
 //  Set block to draw a 5x8 block
 //-------------------------------------------------
 
-int ef9365_device::draw_character( unsigned char c, int block, int smallblock )
+int ef9365_device::draw_character(uint8_t c, bool block, bool smallblock)
 {
-	int x_char,y_char;
-	unsigned int x, y;
-	int x_char_res,y_char_res;
-	int p_factor,q_factor,p,q;
-	int compute_cycles;
+	uint16_t x = get_x_reg();
+	uint16_t y = get_y_reg();
 
-	x = get_x_reg();
-	y = get_y_reg();
+	int x_char_res = 5;
+	int y_char_res = 8;
 
-	x_char_res = 5;
-	y_char_res = 8;
-
-	if( smallblock )
+	if (smallblock)
 	{
-		block = 1;
+		block = true;
 		x_char_res = 4;
 		y_char_res = 4;
 	}
 
-	p_factor = (m_registers[EF936X_REG_CSIZE] >> 4);
-	if(!p_factor)
+	uint8_t p_factor = m_registers[EF936X_REG_CSIZE] >> 4;
+	if (!p_factor)
 		p_factor = 16;
 
-	q_factor = (m_registers[EF936X_REG_CSIZE] &  0xF);
-	if(!q_factor)
+	uint8_t q_factor = m_registers[EF936X_REG_CSIZE] & 0xf;
+	if (!q_factor)
 		q_factor = 16;
 
-	compute_cycles = ( ( x_char_res + 1 ) * p_factor ) * ( y_char_res * q_factor );
+	int compute_cycles = ((x_char_res + 1) * p_factor) * (y_char_res * q_factor);
 
-	if(c<96)
+	if (c < 0x60)
 	{
-		for( x_char=0 ; x_char < x_char_res ; x_char++ )
+		for (int x_char = 0; x_char < x_char_res; x_char++)
 		{
-			for( y_char = y_char_res - 1 ; y_char >= 0 ; y_char-- )
+			for (int y_char = y_char_res - 1; y_char >= 0; y_char--)
 			{
-				if ( block || get_char_pix( c, x_char, ( (y_char_res - 1) - y_char ) ) )
+				if (block || get_char_pix(c, x_char, ((y_char_res - 1) - y_char)))
 				{
-					if( m_registers[EF936X_REG_CTRL2] & 0x04) // Tilted character
+					if (m_registers[EF936X_REG_CTRL2] & 0x04) // Italic mode
```

Comment:
Strictly speaking, this is “oblique”, not “Italic” (true Italic faces use stylised handwriting, oblique faces are formed by skewing a regular face).

Reply:
Replaced the three instances of "Italic" with "Oblique".